src/mapper/pkg/cloudapi/cloud_client.go

```diff
@@ -0,0 +1,16 @@
+package cloudapi
+
+import (
```

Comment:
It's a bad practice in go to put underscores in file names because some suffixes actually do black magics (like `_test.go` and `_linux.go`)